<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerieListViewWidget</class>
 <widget class="QWidget" name="SerieListViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="listMockButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>List mock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSerieButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
